src/ts/components/settingspanel.ts

```diff
@@ -76,7 +78,8 @@ export class SettingsPanel extends Container<SettingsPanelConfig> {
       // Attach marker class to last visible item
       let lastShownItem = null;
       for (let component of this.getItems()) {
-        if (component instanceof SettingsPanelItem) {
+        if ((component instanceof SettingsPanelItem)
+            || (component instanceof SubtitlePanelCloser)) {
```

Comment:
The `SettingsPanel` must not know anything about `SubtitlePanelCloser` (or any other specific components). Please find a generic solution. 

General rule: Components must not have explicit relations with any components except subcomponents that they require for their specific functionality (e.g. `SettingsPanelItem` is a subcomponent of `SettingsPanel`)